### DISCUSSION TITLE
Optional Modul: Managarr

### DISCUSSION THREAD

**Discussion by KnuffigerMaxi** [ahembree/ansible-hms-docker#111] [Ideas]
Hi, 
Here i found this on Reddit: 
[Reddit-Post](https://www.reddit.com/r/selfhosted/s/SwrNIA90Vn
)
Git-Repository [Managarr](https://github.com/Dark-Alex-17/managarr?tab=readme-ov-file)

**Comment by ahembree** [comment]
I just attempted to add this in and was near the end but it did not work correctly when testing.

The container starts and is able to attach successfully to the Sonarr and Radarr instances and interact with them, but once you leave the container using the detach sequence (CTRL+p, CTRL+q), it essentially "breaks" your current terminal session and you have to close and re-open it. If you then attempt to reattach to the Managarr container (using `docker attach managarr`), itself is basically broken and probably has to be restarted.

I don't believe this is an issue with Managarr itself, but rather with Docker Compose and TTY/STDIN. I could be wrong though.

So it is not possible to add this container to the Compose part of the project in a way that I like.
